atsign-foundation/at_dude
# fix: NAMESPACE and APIKEY set in the .env file

**- What I did**

- Changed the namespace from at_skeleton app
- Added an api key for production (located in .env file which is gitignored)
This PR closes #21 

**- How I did it**
**- How to verify it**

1. create .env file if it doesn't exist
2. add NAMESPACE=at_dude in .env file
3. add API_KEY =<your api key>
4. Run the app, you should not see a .env not loaded nor warning set API_KEY before using app in production warning message in the debug console.

**- Description for the changelog**

NAMESPACE and API_KEY set for the dude app
